### PR TITLE
fix ElementwiseKernel.__call__ docs

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix document of ``size`` as variable ``n`` seems no longer be used in generated CUDA source of ElementwiseKernel.

Note that currently ``__call__`` is not exposed to API reference (#458)

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins test this please

File: cupy/core/elementwise.pxi [hunk 1]
```diff
@@ -514,7 +514,9 @@ cdef class ElementwiseKernel:
             raise ValueError("Can not use 'i' as a parameter name")
 
     def __call__(self, *args, **kwargs):
-        """Compiles and invokes the elementwise kernel.
+        """__call__(*args, **kwargs)
```
Comment (niboshi) [inline cupy/core/elementwise.pxi]: What's the purpose of this change?

Comment (kmaehashi) [reply cupy/core/elementwise.pxi]: Removed this change.
I intended to align with `ReductionKernel` but it was meaningless.  Ref. #808

**Comment by niboshi** [comment]
LGTM!
